File: ClientApp/src/app/userAccount.ts
export class userAccount {
    id: number;
    userName: string;
    password: string;
    firstName: string;
    lastName: string;
    contact: string;
    address: string;

    constructor(id, userName, password, firstName, lastName, contact, address) {
        this.id = id;
        this.userName = userName;
        this.password = password;
        this.firstName = firstName;
        this.contact = contact;
        this.address = address;
    }
}

File: Model/UserAccount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace TheThreeCanineCapstoneCodeketeers.Model
{
    public class UserAccount
    {
        [Key]
        public int id { get; set; }
        public string userName { get; set; }
        public string password { get; set; }
        public string firstName { get; set; }
        public string lastName { get; set; }
        public string contact { get; set; }
        public string address { get; set; }

    }
}
